contentcuration/contentcuration/frontend/channelEdit/components/ResourcePanel.vue

```diff
@@ -892,6 +910,8 @@
       noQuestionsError: 'Exercise is empty',
       incompleteQuestionError:
         '{count, plural, one {# incomplete question} other {# incomplete questions}}',
+      sourceContentDoesntExist:
+        'Source content no longer exists. Please contact your administrator.',
```

Comment:
We don't have time to translate this so I think this change should just go into unstable.